<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="minimumSize">
   <size>
    <width>1024</width>
    <height>768</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout"/>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1024</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionDevice_Database"/>
    <addaction name="action_Preferences"/>
    <addaction name="separator"/>
    <addaction name="actionE_xit"/>
   </widget>
   <widget class="QMenu" name="menuWindow">
    <property name="title">
     <string>Window</string>
    </property>
    <addaction name="separator"/>
    <addaction name="action_Driver_Statistics"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuWindow"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionOpen"/>
   <addaction name="action_Close"/>
   <addaction name="separator"/>
   <addaction name="action_AddNode"/>
   <addaction name="action_Delete_Node"/>
   <addaction name="action_Heal_Network"/>
   <addaction name="separator"/>
   <addaction name="actionDevice_Database"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action_Preferences">
   <property name="text">
    <string>&amp;Preferences</string>
   </property>
   <property name="toolTip">
    <string>Preferences</string>
   </property>
  </action>
  <action name="actionE_xit">
   <property name="text">
    <string>E&amp;xit</string>
   </property>
  </action>
  <action name="action_Driver_Statistics">
   <property name="text">
    <string>&amp;Driver Statistics</string>
   </property>
  </action>
  <action name="actionDevice_Database">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>&amp;Device Database</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>&amp;Open</string>
   </property>
   <property name="toolTip">
    <string>Open</string>
   </property>
  </action>
  <action name="action_Close">
   <property name="text">
    <string>&amp;Close</string>
   </property>
   <property name="toolTip">
    <string>Close Local or Remote Connection</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="action_AddNode">
   <property name="text">
    <string>&amp;Add Node</string>
   </property>
  </action>
  <action name="action_Delete_Node">
   <property name="text">
    <string>&amp;Delete Node</string>
   </property>
  </action>
  <action name="action_Heal_Network">
   <property name="text">
    <string>&amp;Heal Network</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>actionE_xit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
